src/tailwind.css

```diff
@@ -72,5 +72,11 @@
   margin: 0 auto; /* center on page */
   display: block; /* required for max-width to kick in */
   width: 100%;
-  max-width: 1280px;
+  max-width: 720px;
```

Comment:
> The max-width of these cards is 720px; does that mean the max width
of the entire calculator should also be 720px, including the form?
(That's what I've implemented here.)

So, I was curious about this as well. The Calc and c
ards don't look as good on really large screens with this set to only 720 px and I could have been looking at the wrong place in the designs, but it looked like at least here...
<img width="750" alt="Screenshot 2024-06-21 at 9 19 32 AM" src="https://github.com/rewiringamerica/embed.rewiringamerica.org/assets/1214752/71ae804a-cca9-49a3-9b54-33870a50909b">

That the cards should be allowed to be larger.

Reply:
Confirmed in DMs with Emiola that the whole thing is actually supposed to be 720px max-width. (I think you're looking at an outdated part of the Figma -- the part I'm working from has gotten rid of the pill button project selector.)

I'm going to update the demo page to match that max width.